apis/apps/v1alpha1/componentdefinition_types.go

```diff
@@ -948,6 +948,39 @@ type Action struct {
 	PreCondition *PreConditionType `json:"preCondition,omitempty"`
 }
 
+type Probe struct {
+	Action `json:",inline"`
+
+	// TODO: remove this later.
+	//
+	// +optional
+	BuiltinHandler *BuiltinActionHandlerType `json:"builtinHandler,omitempty"`
+
+	// Specifies the number of seconds to wait after the container has started before the RoleProbe
+	// begins to detect the container's role.
```

Comment:
Is InitialDelaySeconds only applicable to roleProbe?

Reply:
It's applicable to all probes.